### PR TITLE
Add Vertex Images Support To Line Chart

### PR DESCRIPTION
**Pull request by kurtenjoybloom**
## Description
Update the LineChart to support images on vertices.

[Ticket](https://www.notion.so/enjoybloom/Playground-for-New-Charts-Library-9f8c40db681449e79ccf75b54fe16129)

&nbsp;

## Review Checklist
_A view reminders on what to check when submitting a PR._

- [ ]  Tracking added.
- [ ]  Naming conventions.
- [ ]  Linting throws no warnings or errors.
- [ x ]  Tests are running locally.
- [ ]  Feature flags are configured correctly.
- [ x ]  Can this PR create a major harm? If so, flag accordingly.

### CODE REVIEW

**Comment by misolubarda** [comment]
It took me a while before I figured out that this is a new project 😅

**Review by misolubarda — APPROVED** [review]

**Comment by misolubarda** [comment]
@kurtenjoybloom I approved the PR. Feel free to introduce changes or just let it be as it is.

File: Array+Access.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//  Array+Access.swift
+//  Charts
+//
+//  Created by Kurt Jacobs on 2022/03/18.
+//
+
+import Foundation
+
+extension Collection {
+    subscript (safe index: Index) -> Element? {
```
[the remark this answers is not in the document]
Comment (kurtenjoybloom) [reply Array+Access.swift]: fixed

**Review by misolubarda — APPROVED** [review]

File: Charts.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -413,19 +421,8 @@
 		2227EA581FF1F224007D98D9 /* ChartRenderers */ = {
 			isa = PBXGroup;
 			children = (
-				0216EDC6A1FE272F4EB19FCF /* DataRenderer.swift */,
```
[the remark this answers is not in the document]
Comment (kurtenjoybloom) [reply Charts.xcodeproj/project.pbxproj]: fixed!

**Review by misolubarda — APPROVED** [review]

**Pull request merged**